src/main/java/com/bootcamp/demo/model/Session.java

```diff
@@ -0,0 +1,48 @@
+package com.bootcamp.demo.model;
+
+import java.time.LocalDateTime;
+
+/**
+ * A Session object stores the user, the start and end date and time of the session
+ *
+ * Next needed update: DI of the User attribute automatically when a session is created, and the user will be
+ * recognised after the Log In
+ *
+ * @author Denisa Dragota
+ * @version 11/11/2021
+ */
+public class Session {
+
+    private final LocalDateTime startSession;
+    private LocalDateTime endSession;
+    private User user;
+
+    /**
+     * Session constructor that initialises a session start time with the current date and time
+     * when creating the object and setting the end date and time null
+     */
+    public Session(){
```

Comment:
**Important for all team members:** pay special attention to proper code formatting in Java. 
For example, the proper code formatting for this constructor should have been:
`public Session() {` (with a space between the parentheses and the accolade)
There are some other details as well in this file. 

Always ensure you have your files properly formatted before committing them. One way to do that is to have the class opened in Intelijj,  select from the Intellij menu Code -> Reformat code (or use the keyboard shortcut).
In time, this will become a standard reflex: to format the file, optimise imports and save the file frequently.